docs/site.json

```diff
@@ -14,6 +14,18 @@
     },
     {
       "glob": "userGuide/*.md"
+    },
+    {
+      "src": "userGuide/fullSyntaxReference.md",
+      "searchable": "no"
```

Comment:
It prunes everything off though. So now this is not searchable at all, whereas previously it will just appear as two separate entries (which is not desirable but at least still searchable?)

Reply:
It is not a big loss because all content appear elsewhere, which I consider to be their 'proper' location. It doesn't make sense to have two results for the exact same content. This page is not a good destination for search results due to all the dynamic panels (the user ends up at the wrong scroll position in the page due to how the page is loaded in stages). Unfortunately, this page appears at the top of the search results, above the more preferred twin result. Let's get rid of it for now. We can bring it back if we 'miss it' after a period of dog fooding.